app/views/shared/flashes/upstream-webinar-1.html.erb

```diff
@@ -0,0 +1,8 @@
+Live webinar: How is Bloomberg supporting the Python community by shifting left? Join us 8/4. 
```

Comment:
Worth adding the year to that date?

Reply:
We swap them out before it matters (I just get the content from Amy :)